Makefile

```diff
@@ -27,5 +27,6 @@ e2e:
 
 .PHONY: coverage
 coverage:
-	go test -covermode=count -coverprofile=profile.cov $(shell go list ./... | grep -v /vendor/)
-	go tool cover -func=profile.cov
+	@go test -covermode=count -coverprofile=profile.cov.tmp -coverpkg ./pkg/... $(shell go list ./... | grep -v /vendor/)
+	@cat profile.cov.tmp | grep -v "_k8s.go" > profile.cov
```

Comment:
It would be better not to create a tmp file.

```
@go test -covermode=count -coverprofile=profile.cov -coverpkg ./pkg/... $(shell go list ./... | grep -v /vendor/)
@sed -i "/_k8s.go/d" profile.cov
```

.. or simply `@rm profile.cov.tmp`.

Reply:
`sed` behaves differently in OSX and Linux. I'd rather just remove the tmp file.